airflow/providers/google/provider.yaml

```diff
@@ -83,6 +83,7 @@ dependencies:
   - google-cloud-build>=3.0.0
   - google-cloud-compute>=0.1.0,<2.0.0
   - google-cloud-container>=2.2.0,<3.0.0
+  - google-cloud-dataflow-client==0.5.4
```

Comment:
Can you not pin it to a specific version please, and instead provide a range or just min version ideally.

Airflow is a library as well as app so we want to keep deps as open as possible.